actioncable/lib/action_cable/connection/client_socket.rb

```diff
@@ -29,10 +29,11 @@ def self.secure_request?(env)
 
       attr_reader :env, :url
 
-      def initialize(env, event_target, event_loop)
+      def initialize(env, options, event_target, event_loop)
```

Comment:
Would be happy to do this as it is cleaner. I haven't seen much use of the Ruby keyword arguments in the Rails codebase, but I guess it's ok now?